src/transforms/extensions.ts

```diff
@@ -20,9 +20,12 @@ import { cloneDeep } from "lodash";
  * This may result in additional operations being inserted into the model.
  * @param codeModel The model that contains all the information required to generate a service API.
  */
-export function normalizeModelWithExtensions(codeModel: CodeModel) {
+export function normalizeModelWithExtensions(
+  codeModel: CodeModel,
+  ignorePageableArrayCheck: boolean
```

Comment:
this feels like an unnecessarily pervasive change. I am not very familiar with the code base, do not we have an options bag to house this option? cc @joheredi

Reply:
Not as of now. But, I was planning to take that task later this month